### ISSUE TITLE
CloudDatabases not loaded properly

### ISSUE THREAD

**Issue by waltlenu** [rackerlabs/rackspace-sdk-plugin#20]
I think something is still not quite right since `openstacksdk` v0.7.1. The plugin still doesn't get overridden. Testing the `rackspacesdk` now from a new Virtualenv:

``` python
>>> from rackspace import connection
>>> conn = connection.Connection(username="", api_key="", region="")
>>> len(list(conn.database.instances()))
7
>>> list(conn.database.backups())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Proxy' object has no attribute 'backups'
>>> conn.database.__module__
'openstack.database.v1._proxy'
```

Enabling logging:

``` python
utils.enable_logging(debug=True, stream=sys.stderr)
```

... just results in a similar line:

```
2015-12-01 15:20:10,370 DEBUG: openstack.profile Overriding object-store with {'service_type': 'object-store', 'interface': None, 'version': None, 'region_name': None, 'service_name': None}
```

I used the following steps to create a new virtual environment (wall of text, I know, but all package versions are there):

``` bash
# python --version
Python 2.7.10
# virtualenv --version
13.1.2
# virtualenv test
New python executable in test/bin/python2.7
Also creating executable in test/bin/python
Installing setuptools, pip, wheel...done.
# pip --version
pip 7.1.2 from ...
# cd test; source bin/activate
(test)
# pip install rackspacesdk
Collecting rackspacesdk
  Using cached rackspacesdk-0.4.0-py2.py3-none-any.whl
Collecting six (from rackspacesdk)
  Using cached six-1.10.0-py2.py3-none-any.whl
Collecting rackspaceauth==0.2.0 (from rackspacesdk)
  Using cached rackspaceauth-0.2.0-py2.py3-none-any.whl
Collecting openstacksdk>=0.7.1 (from rackspacesdk)
  Using cached openstacksdk-0.7.1-py2.py3-none-any.whl
Collecting pbr>=1.6 (from rackspaceauth==0.2.0->rackspacesdk)
  Using cached pbr-1.8.1-py2.py3-none-any.whl
Collecting keystoneauth1>=1.0.0 (from rackspaceauth==0.2.0->rackspacesdk)
  Using cached keystoneauth1-1.2.0-py2.py3-none-any.whl
Collecting requests!=2.8.0,>=2.5.2 (from rackspaceauth==0.2.0->rackspacesdk)
  Using cached requests-2.8.1-py2.py3-none-any.whl
Collecting oslo.utils>=2.8.0 (from openstacksdk>=0.7.1->rackspacesdk)
  Using cached oslo.utils-3.0.0-py2.py3-none-any.whl
Collecting stevedore>=1.5.0 (from openstacksdk>=0.7.1->rackspacesdk)
  Using cached stevedore-1.9.0-py2.py3-none-any.whl
Collecting os-client-config!=1.6.2,>=1.4.0 (from openstacksdk>=0.7.1->rackspacesdk)
  Using cached os_client_config-1.12.0-py2.py3-none-any.whl
Collecting argparse (from keystoneauth1>=1.0.0->rackspaceauth==0.2.0->rackspacesdk)
  Using cached argparse-1.4.0-py2.py3-none-any.whl
Collecting iso8601>=0.1.9 (from keystoneauth1>=1.0.0->rackspaceauth==0.2.0->rackspacesdk)
  Using cached iso8601-0.1.11-py2.py3-none-any.whl
Collecting monotonic>=0.3 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
Collecting oslo.i18n>=1.5.0 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached oslo.i18n-3.0.0-py2.py3-none-any.whl
Collecting netifaces>=0.10.4 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
Collecting netaddr!=0.7.16,>=0.7.12 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached netaddr-0.7.18-py2.py3-none-any.whl
Collecting debtcollector>=0.3.0 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached debtcollector-1.0.0-py2.py3-none-any.whl
Collecting Babel>=1.3 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached Babel-2.1.1-py2.py3-none-any.whl
Collecting pytz>=2013.6 (from oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached pytz-2015.7-py2.py3-none-any.whl
Collecting PyYAML>=3.1.0 (from os-client-config!=1.6.2,>=1.4.0->openstacksdk>=0.7.1->rackspacesdk)
Collecting appdirs>=1.3.0 (from os-client-config!=1.6.2,>=1.4.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached appdirs-1.4.0-py2.py3-none-any.whl
Collecting requestsexceptions>=1.1.1 (from os-client-config!=1.6.2,>=1.4.0->openstacksdk>=0.7.1->rackspacesdk)
  Using cached requestsexceptions-1.1.1-py2-none-any.whl
Collecting wrapt>=1.7.0 (from debtcollector>=0.3.0->oslo.utils>=2.8.0->openstacksdk>=0.7.1->rackspacesdk)
Installing collected packages: six, pbr, argparse, stevedore, iso8601, requests, keystoneauth1, rackspaceauth, monotonic, pytz, Babel, oslo.i18n, netifaces, netaddr, wrapt, debtcollector, oslo.utils, PyYAML, appdirs, requestsexceptions, os-client-config, openstacksdk, rackspacesdk
Successfully installed Babel-2.1.1 PyYAML-3.11 appdirs-1.4.0 argparse-1.4.0 debtcollector-1.0.0 iso8601-0.1.11 keystoneauth1-1.2.0 monotonic-0.4 netaddr-0.7.18 netifaces-0.10.4 openstacksdk-0.7.1 os-client-config-1.12.0 oslo.i18n-3.0.0 oslo.utils-3.0.0 pbr-1.8.1 pytz-2015.7 rackspaceauth-0.2.0 rackspacesdk-0.4.0 requests-2.8.1 requestsexceptions-1.1.1 six-1.10.0 stevedore-1.9.0 wrapt-1.10.5

```

**Comment by briancurtin** [comment]
Confirmed. Looking into it right now.

**Comment by briancurtin** [comment]
This should be fixed by the work in #37, which was released in rackspacesdk 0.7.5: https://pypi.python.org/pypi/rackspacesdk/0.7.5
